5_kata.py

```diff
@@ -0,0 +1,14 @@
+#https://www.codewars.com/kata/58f6000bc0ec6451960000fd/train/python
+
+def sel_reverse(arr,l):
+    if l>0 and len(arr) > l:
```

Comment:
Please add spaces before and after mathematical operator. The same way as you have it in `len(arr) > l`.